cornerstone_widget/cs_widget.py

```diff
@@ -108,8 +109,69 @@ def get_tool_state(self):
             return {}
 
     def set_tool_state(self, state):
+        # type: (Dict) -> None
         """A method for feeding data into the widget"""
         self._tool_state_in = json.dumps(state)
+        self._tool_state_out = json.dumps(state)
```

Comment:
why the two tool states?

Reply:
so this isn't a very elegant approach, but one is for setting the state (_in) and one is for reading the state (_out) since the state itself isn't a traitlet and the traitlet is only updated on a mouseup event